src/lib/help.ts

```diff
@@ -20,10 +20,44 @@ export default class RelayHelp extends Help {
     super.showHelp(args)
   }
 
+  protected async customShowRootHelp(): Promise<void> {
+    let rootTopics = super.sortedTopics
+    let rootCommands = super.sortedCommands
+
+    const state = this.config.pjson?.oclif?.state
+    if (state) {
+      super.log(state === `deprecated` ? `${super.config.bin} is deprecated` : `${super.config.bin} is in ${state}.\n`)
+    }
+
+    super.log(super.formatRoot())
+    super.log(``)
+
+    // take out top level help for capsule commands
+    rootTopics = rootTopics.filter((t) => !t.name.includes(`task`))
```

Comment:
Seems better if this can be managed via config or within the command impls.

I found something that works:  https://github.com/relaypro/relay-cli/pull/157